ckanext/activity/model/activity.py

```diff
@@ -320,8 +342,9 @@ def _to_list(vals: Union[List[str], Tuple[str], str]):
 
 def _package_activity_query(package_id: Union[str, List[str]]) -> QActivity:
     """Return an SQLAlchemy query for all activities about package_id."""
-    q = model.Session.query(Activity).filter(
-        Activity.object_id.in_(_to_list(package_id)))  # type: ignore
+    q = model.Session.query(Activity)
+    q = q.options(_activities_defer_data())
```

Comment:
Could use backslashes to reduce unnecessary assignments.

    q = model.Session.query(Activity)\
        .options(_activities_defer_data())\
        .filter(...